Tweak.x

```diff
@@ -228,12 +230,16 @@ void log_impl(NSString *logStr) {
 		LOG(@"Couldn't send identifiers: %@", sendErr);
 }
 
-- (void)writeToStateWithCode:(NSString * __nullable)code
+- (void)saveStateWithCode:(NSString * __nullable)code
                       secret:(NSString * __nullable)secret
 				   connected:(BOOL)connected
 				       error:(NSError * __nullable)error {
+	currentState = [BPState.alloc initWithCode:code secret:secret connected:connected error:nil];
```

Comment:
It seems like this is an error left over from my changes, but I'm guessing `error` is not supposed to be always nil. Does that seem correct to you as well?
```suggestion
	currentState = [BPState.alloc initWithCode:code secret:secret connected:connected error:error];
```

Reply:
Seems to me like that should not be nil too, I made a commit with your suggestion.